deploy/helm/charts/templates/zfs-contoller.yaml

```diff
@@ -38,6 +38,13 @@ spec:
       priorityClassName: {{ template "zfslocalpv.zfsController.priorityClassName" . }}
 {{- end }}
       serviceAccount: {{ .Values.serviceAccount.zfsController.name }}
+{{- if .Values.zfsController.initContainers }}
```

Comment:
Should add initContainers and additionalVolumes to values.yaml with `[]` default.